_includes/team.html

```diff
@@ -0,0 +1,24 @@
+<div id="team">
+    <div class="section-white top-section-border">
+        <div class="container">
+            <div class="row">
+            <img src="{{site.url}}/assets/globe-model.svg" class="section-icon img-responsive" alt="steering committee icon">
+                <h3 class="col-sm-12 section-header">Core Developers</h3>
+                <p class="col-sm-12 support-paragraph">
+                    (in alphabetical order).
+                </p>
+            </div>
+        </div>
+    </div>
+  <div class="col-md-12 section-content section-center">
+      {% assign sorted = site.data.core.coredevs | sort: "last" %}
+      {% for member in sorted %}
+      <div class="col-md-4 material">
+        <img class="core-member-photo" alt="{{member.first}} {{member.last}}'s avatar picture" src='{{member.avatar}}' />
+        <p class="card-header">{{member.first}} {{member.last}}</p>
```

Comment:
to solve the formatting issue maybe a solution could be to use `</br>` for the next line instead of a new `<p`?